lib/src/source/hosted.dart

```diff
@@ -151,21 +151,28 @@ class HostedSource extends Source {
   }
 }
 
+class _VersionInfo {
```

Comment:
Can we give this a one-liner comment, something like:
`/// Information about a package version as served from pub server`

Reply:
Done